### DISCUSSION TITLE
how to change pnpm store folder path

### DISCUSSION THREAD

**Discussion by mhamri** [pnpm/pnpm#7289] [Q&A]
with introduction of [dev drive in windows ](https://learn.microsoft.com/en-us/windows/dev-drive/)

it's suggested to move the store and cache folder into the dev drive. 

how can I achieve that, also I think some update to the documentation about this would be nice

**Comment by jilherme** [comment]
did you figured it out?

**Comment by zkochan** [comment]
You can't move just the store to dev drive. You have to develop on the dev drive. You should clone your repository to the dev drive. The store will automatically be created on the dev drive.

**Reply by jilherme** [reply]
yeah, yesterday I finally decided to move my projects from C: to the dev drive and found the answers in the FAQ. Thanks for your time!
